### DISCUSSION TITLE
Is there any system RAM improvement with forge?

### DISCUSSION THREAD

**Discussion by infinitedusky** [lllyasviel/stable-diffusion-webui-forge#383] [Q&A]
Automatic1111 uses over 100G on my machine. I feel like there must be a management issue that causes this. It's like this when the no inference is happening.

**Comment by Razgrits** [comment]
Ditto, it used to not be like this, I think it has to do with torch 2.1.2. After a few generations I constantly run out of system memory AND vram. It makes it impossible to use Forge. It completely obliterates both my vram, shared memory and committed. And it doesn't matter how much paging mem I put in, it will constantly fill it without fail. Sad to see this is a month old and no one has replied.
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/45423559/0919caef-5641-4071-a3f9-2888dd05a938)
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/45423559/2ccb545e-3bce-4470-906c-b1f6854c4adb)

I think it has to do with LoRA and Torch but there is literally no info on it anywhere.
